src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -266,7 +269,7 @@ class SidebarPanels extends React.Component<Props, State> {
                                     hasSidebarInitialized={isInitialized}
                                     ref={this.boxAISidebar}
                                     startMarkName={MARK_NAME_JS_LOADING_BOXAI}
-                                    userInfo={{name: currentUser?.name, avatarUrl: currentUser?.avatar_url}}
+                                    userInfo={{ name: currentUser?.name, avatarURL: currentUser?.avatar_url }}
```

Comment:
looks like there's a spacing issue

Reply:
It seems that earlier we must have merged something with incorrect formatting, as this is already the second time we're fixing spacing and commas